style.css

```diff
@@ -0,0 +1,231 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans&display=swap');
+
+*{
+    font-family: 'Noto Sans', sans-serif;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+:root{
+    --bg-color: #F3F4F6;
+    --primary-color: #226A80;
+    --white-color: #FFF;
+}
+
+section{
+    position: relative;
+    width: 100%;
+    min-height: 100vh;  
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    background-color: var(--bg-color);
+}
+
+.hero{
+    position: relative;
+    width: 16rem;
+    bottom: 16rem;
+    left: 80rem;
+}
+
+header{
+    position: relative;
+    top: 0;
+    width: 100%;
+    padding: 1.875rem 6.25rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+header .logo{
+    position: relative;
+    color: #000;
```

Comment:
Just drawing your attention, color is hard-coded here.

Reply:
Hey, thanks for approving the changes! For now, some of the colors are indeed temporary, and thanks for bringing up the usage of 'rem'. Will make the necessary changes.